setup.py

```diff
@@ -46,8 +46,8 @@
             "upgrade PyPy to use this library."
         )
 else:
-    requirements.append("cffi>=1.7.0")
-    setup_requirements.append("cffi>=1.7.0")
+    requirements.append("cffi>=1.4.1")
+    setup_requirements.append("cffi>=1.4.1")
```

Comment:
I think this could help in correctly solving #186 insetead of working around the problem like in #187